Makefile

```diff
@@ -45,6 +44,15 @@ ifeq ($(ARCH),arm)
                 USERCFLAGS += -march=armv7-a
         endif
 
+        ifeq ($(ARMV),8)
+                # Untested hack to run the arm32 bit test on Travis aarch64 with 'setarch linux32'.
+                # Running 'setarch linux32 uname -m' returns armv8l on travis aarch64.
+                USERCFLAGS += -march=armv7-a
+                ARMV := 7
+        endif
+
+        DEFINES		:= -DCONFIG_ARMV$(ARMV) -DCONFIG_VDSO_32
+
```

Comment:
I think it makes sense - it's in reality not armv7, but aarch32 which is instruction-compatible with first.
My only concern would be that vdso on arm32 differ from compatible vdso on aarch64, but so far we check only build - it shouldn't make any difference.

Please, change the commentary from `Untested hack to run the arm32` to something more appropriate after Travis-CI passes.

Reply:
> Please, change the commentary from `Untested hack to run the arm32` to something more appropriate after Travis-CI passes.

Travis-CI has already passed. Should I change it before or after merge?

Reply:
Whatever is more suitable - probably a patch on the top that just changes the comment won't need much testing..

Reply:
Will do it in a follow up.